kornia/feature/defmo.py

```diff
@@ -0,0 +1,112 @@
+from typing import Dict
+
+import torch
+import torch.nn as nn
+import torchvision.models
+
+urls: Dict[str, str] = {}
+urls["defmo_encoder"] = "http://ptak.felk.cvut.cz/personal/rozumden/defmo_saved_models/encoder_best.pt"
+urls["defmo_rendering"] = "http://ptak.felk.cvut.cz/personal/rozumden/defmo_saved_models/rendering_best.pt"
+
+
+class EncoderDeFMO(nn.Module):
+    def __init__(self):
+        super(EncoderDeFMO, self).__init__()
+        model = torchvision.models.resnet50(pretrained=True)
```

Comment:
One question here, shall we allow more models than resnet50? Probably adding those ```timm``` models avaliable as well?

Reply:
@shijianjian currently not, because kornia does not depends on timm. While we may need to find some solution for handling pretrained models with a standard backbones, I believe, it is not related to this particular PR

Reply:
The idea is to provide curated methods/models. /cc @ethanrublee @garybradski

If the paper states that resnet50 gives average best performance that’s what I would use. Later if a user wants to explore other options, since everything it’s exposed as a module, with python should be pretty straightforward to plug any model and fine tune. BTW, @rozumden do you have somewhere training code for this model ?

Reply:
great - sure we can define later a proper fine tuning api